src/Admin/Vue/PaymentMethods.php

```diff
@@ -6,14 +6,20 @@
 
 class PaymentMethods {
 
+	private $options;
+
+	public function __construct($options) {
+		$this->options = $options;
```

Comment:
if ( empty($options){
$this->options = get_options
}else{
$this->options = $options
}

Reply:
done